integration-tests/tests/pg.test.ts

```diff
@@ -1562,3 +1562,22 @@ test.serial('prefixed table', async (t) => {
 
 	await db.execute(sql`drop table ${users}`);
 });
+
+test.serial('timestamp timezone', async (t) => {
```

Comment:
Did you actually run this test? We don't run the integration tests in the pipeline yet, so it's up to the developers to run them locally. This one looks like it won't work, because you're not creating the "users" table, you're just defining it, and I'm pretty sure there's already a different "users" table defined in the tests and present in the test DB.

Reply:
The test succeeded, but I can see now that it doesn't make any sense. It seems I wasn't paying good attention writing this part 😅. Thanks for pointing it out!
I pushed a fix where a new table is actually created, also with a different name.